helm/config/log_format.conf

```diff
@@ -6,42 +6,51 @@ log_format main escape=json
 '{'
     '"Timestamp":$nanosec,'
     '"Attributes":{'
-        '"mapcolonies.time_local":"$time_local",'
+        '"mapcolonies.time_local": "$time_local",'
         {{ if .Values.authorization.enabled }}
-        '"mapcolonies.http.auth.token.client_name":"$jwt_payload_sub",'
+        '"mapcolonies.http.auth.token.client_name": "$jwt_payload_sub",'
```

Comment:
@CptSchnitz , back in the days we tried to put mapcolonies custom fields under "mapcolonies" key, do  you think we should put it in the body or message key ?